core/app/profile.go

```diff
@@ -70,6 +72,12 @@ func applyProfiler(ctx context.Context, flags *ProfileFlags) func() {
 			log.I(ctx, "Trace profile written")
 		})
 	}
+	if flags.Pprof {
+		log.I(ctx, "Enabling pprof on :6060")
+		go func() {
+			http.ListenAndServe("localhost:6060", nil)
```

Comment:
is it possible that the cmd will be run with multiple instance

Reply:
You technically COULD, but its unlikely.
EVen if you do it with gapit, you would have to 

gapit -profile-pprof -gapis-args -profile-pprof

Which is probably .... less useful.

Reply:
As we're binding to port 6060 here, so that assume that it can be only with one instance, which is not like other commands such as  `gapit report`, you can run multiple of `gapit report` at the same time.

Reply:
This isn't a verb though, this is a flag that can run to ANY/all applications. It would like:
gapit -profile-pprof report ......
You just cannot run 2 applications with the same arg at the same time. 
We could make gapit -profile-pprof 6060 report, but that is a lot more typing, since I don't expect users to use the hidden flags that way.

(remember these flags are hidden)